README.md

```diff
@@ -1,30 +1,36 @@
 # Kafka Connect HTTP Connector
```

Comment:
This whole file has been reformatted, which makes it harder to review the diff, and breaks some criteria previously followed, like use of `*` and `-`.

I think that should be respected, and only the diff should appear here.

Reply:
Looks like it got formatted because of the default formatter of my editor. I have reverted it. Please have a look.